### PR TITLE
Edit Linux Text

### PR DESCRIPTION
**Pull request by tuanhxh1**
Detects suspicious commands for edit text using sed

### CODE REVIEW

**Comment by nasbench** [comment]
I removed the `sed` rule since it doesn't look for anything suspicious and the PR was inactive so we didn't get any information from the author. 

I kept the `usermod` rule since it's useful and I updated it to fit SIGMA convetions.

It's ready to merge.

**Review by tuanhxh1 — COMMENTED** [review]
Look good

**Comment by nasbench** [comment]
> Look good

Glad you're alive @tuanhxh1 :D 

We will merge this today

**Review by frack113 — APPROVED** [review]

File: rules/linux/process_creation/proc_creation_lnx_usermod_susp_group.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: User Added To Root/Sudoers Group Using Usermod
+id: 6a50f16c-3b7b-42d1-b081-0fdd3ba70a73
+status: test
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_usermod_susp_group.yml:3]: new rules are ``experimental`

Comment (nasbench) [inline rules/linux/process_creation/proc_creation_lnx_usermod_susp_group.yml:3]:
```suggestion
status: experimental
```

**Pull request merged**